community/io/src/test/java/org/neo4j/adversaries/pagecache/AdversarialReadPageCursor.java

```diff
@@ -22,7 +22,11 @@
 import java.io.File;
 import java.io.FileNotFoundException;
 import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
 import java.util.Objects;
+import java.util.concurrent.ThreadLocalRandom;
```

Comment:
Since now inconsistency generation logic is not that plain, It would be great to have updated java doc for this class that will briefly describe inconsistency preparation/read cycle.